src/components/EntityInfo.vue

```diff
@@ -13,7 +13,16 @@
             </div>
 
             <!-- Description -->
-            <div v-if="businessId" id="business-subtitle">{{ businessDescription }}</div>
+            <div v-if="businessId">
+              <span id="business-subtitle">{{ businessDescription }}</span>
+              <v-chip
+                v-if="isAuthorizedToContinueOut"
+                id="authorized-to-continue-out-chip"
+                class="primary mt-n1 ml-3 pointer-events-none" small label text-color="white"
+              >
```

Comment:
EntityInfo.vue includes a large number of `v-if` conditionals.  Instead of adding another `v-if` what do you think about refactoring this component?   For example, it looks like the elements visible for entities with a businessId could be separated from those with a temporary registration number.

Reply:
1. I have another PR going in for this ticket so I can address some of these comments there.

2. I am wary of conditional chaining because it can hide/mask some issues that we should be handling a better way. In this case, however, since the data came from an API, I think it's OK to use CC, especially since we check the result on the next line. There may be another way to handle this as well (eg, when the data is fetched, validate that the received object is complete). I don't think it's always an anti-pattern. Thoughts? Ideas on improving this in general?

3. The conditional display is interspersed in the template. Probably we'd get lots of duplication if we split this apart, and it might cause future issues if we update the business code without updating the other code (or vice-versa). Thoughts?

Reply:
> 3. The conditional display is interspersed in the template. Probably we'd get lots of duplication if we split this apart, and it might cause future issues if we update the business code without updating the other code (or vice-versa). Thoughts?

I don't see a clean way to separate out the conditional sections in EntityInfo without complicating things. I'm thinking of leaving this one alone.